README.md

```diff
@@ -54,7 +54,7 @@ For other installation methods, see: <https://github.com/swagger-api/swagger-cod
 ### 5. Generate the API Client
 
 ```shell
-swagger-codegen generate -i https://api.globalforester.com/swagger/v2/swagger.json -o Swagger -l python
+swagger-codegen generate -i https://api.globalforester.com/swagger/v3/swagger.json -o Swagger -l python
```

Comment:
Update text about versions at the top of this file?

Reply:
Updated and added som information about previous api versions